### PR TITLE
Newsletter for September 2019

### PR DESCRIPTION
**Pull request by Nyco**
My first attempt :-/
Please review carefully...

### CODE REVIEW

**Comment by winfried** [comment]
love it, well done

File: content/posts/newsletter/2019-09-03.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+### Servers
+
+[Prometheus XMPP Blackbox Exporter](https://github.com/horazont/prometheus-xmpp-blackbox-exporter) (Apache license 2.0) allows you to probe XMPP services and export metrics from the probes to Prometheus.
+
+The igniterealtime community has announced multiple releases:
+
+* [Openfire 4.4.1](https://discourse.igniterealtime.org/t/openfire-4-4-1-release/85857)
+* [Monitoring Service 1.8.0 and inVerse 4.2.0.1 plugins](https://discourse.igniterealtime.org/t/monitoring-service-1-8-0-and-inverse-4-2-0-1-plugins-released/85601)
+* [Smack 4.4.0-alpha2](https://discourse.igniterealtime.org/t/smack-4-4-0-alpha2-released/85670)
```
Comment (guusdk) [inline content/posts/newsletter/2019-09-03.md]: Smack is a library. This release should probably not be listed under 'servers'.

File: content/posts/newsletter/2019-09-03.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+url: 2019/09/newsletter-03-september/
+date: 2019-09-03 09:00
+author: nyco
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the September edition of the XMPP Newsletter!
+
+Loads of news for these two months of July and August: two events in Lyon and Stockholm (more to come), new tools, new server releases (Openfire, ejabberd), lots of client releases (Kaidan, Salut à Toi, Xabber Android, Movim, Converse, Beagle IM and Siskin IM). We also experiment a new section about XEP, or specifications. The XMPP/Jabber community is alive and kicking!
```
Comment (guusdk) [inline content/posts/newsletter/2019-09-03.md]: > The XMPP/Jabber community is alive and kicking!

I'd remove this line. By leaving it in, the suggestion is created that we need to tell this to people. :)

File: content/posts/newsletter/2019-09-03.md (hunk 1 above)
Comment (Nyco) [reply content/posts/newsletter/2019-09-03.md]: Done, thx.

File: content/posts/newsletter/2019-09-03.md (hunk 2 above)
Comment (Nyco) [reply content/posts/newsletter/2019-09-03.md]: Done, thanks!

**Pull request merged**